### DISCUSSION TITLE
Cleaning up the log laravel 11

### DISCUSSION THREAD

**Discussion by ManarangID** [spatie/laravel-activitylog#1306] [Q&A]
hi all, please help to make scheduler to run the clean up in lavarel 11 , regards

**Comment by misakstvanu** [comment]
Everything is prepared for what you need. Put the cleanup command to your scheudler:
1. in app/Console/Kernel.php for Laravel < 11
2. in bootstrap/app.php for Laravel 11+

`$schedule->command('activitylog:clean')->daily();`

https://laravel.com/docs/11.x/scheduling

**Comment by ManarangID** [comment]
Thanks

Pada Sel, 24 Sep 2024 18:51, Michal Štván ***@***.***>
menulis:
